win-odbc64/security_dll/native/source/secpwd.cpp

```diff
@@ -217,8 +221,25 @@ SecPwd::SecPwd(const char *dir, const char* fileName,
     if(stat(certDir,&st) != 0)
         throw SecurityException(DIR_NOTFOUND, (char *)certDir);
 
-    certFile = buildName(certDir, fileName, serverName, CER);
-    activeCertFile = buildName(certDir, activeFileName, serverName, ACTIVE_CER);
+    if (lcid == 0x804) // if local charset is not utf8
```

Comment:
Looking at https://msdn.microsoft.com/en-us/library/cc233965.aspx, it appears 0x804 = language tag zh-CN, with default sort order. Do we know that all other locales use UTF-8? (Perhaps the comment on this line should be, "if locale ID is zh-CN")

Reply:
I will change "is not utf8" to "is gbk". Do you know how many character sets we need?

Reply:
No... I suppose we'll let the marketplace tell us.